### DISCUSSION TITLE
Show difficulties's song if each have a different song

### DISCUSSION THREAD

**Discussion by SK-Fast** [ppy/osu#20191] [Ideas] [answered]
So I was playing around with the visualizer, then I noticed about the Nanahira's Collection beatmap on the jukebox menu, There aren't much song as It was because It's only picking one. It would be nice if jukebox list shows all of songs in the beatmap.

Image of the Beatmap difficulties, each one have different song.
![image](https://user-images.githubusercontent.com/60780448/189105767-f822df86-614d-4dc8-93c4-1f70672bde7a.png)
On the jukebox menu(It only show one song!):
![image](https://user-images.githubusercontent.com/60780448/189105977-6e71cf8d-288e-4b99-b6fa-9f4b5971e296.png)

also, IDK which category fit best for this, I mean it isn't a bug neither so.

**Comment by peppy** [comment, marked as answer]
This is out of scope. Beatmaps are not supposed to be collections of tracks.
